src/openms/include/OpenMS/FILTERING/DATAREDUCTION/Deisotoper.h

```diff
@@ -94,6 +98,7 @@ class OPENMS_DLLAPI Deisotoper
             bool annotate_charge = false,
             bool annotate_iso_peak_count = false,
             bool use_decreasing_model = true,
+            unsigned int start_intensity_check = 2,
             bool add_up_intensity = false);
```

Comment:
@enetz if you add new default arguments, please always add at the end to prevent the true/false of another fxn to be interpreted as `start_intensity_check`